lib/controllers/users.js

```diff
@@ -30,12 +30,11 @@ module.exports = Router()
       next(error);
     }
   })
-  // TODO: This route should only be accessible to signed in users
-  .get('/me', (req, res) => {
+  .get('/me', authenticate, async (req, res) => {
+
     res.json(req.user);
   })
-  // TODO: This route should only be accessible to the admin user
-  .get('/', async (req, res, next) => {
+  .get('/', authenticate, authorize, async (req, res, next) => {
```

Comment:
this works too but try to put inside an array instead `[authenticate, authorize]`